submissions/BlueLamantine/js-dom/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Cat & Coffee</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <header id="topHeader">
+      <h1 class="mainHeader">Cat & C<span class="o"></span>ffee</h1>
+    </header>
+    <main class="content">
+      <nav>
+        <button id="menu-btn" type="button" class="icon">
+          <i class="fa fa-bars fa-2x"></i>
+        </button>
+        <ul id="mainMenu" class="menu"></ul>
+      </nav>
+      <div class="coffee">
+        <div id="coffeeCups" class="coffe-content"></div>
```

Comment:
I'd recommend combining these two blocks into one. It'll simplify your rendering logic and reduce the number of layout blocks.

Reply:
@MaxKalashnyk Please, can you specify which exactly blocks need to be combined ? All this blocks ```<div>``` for providing a more efficient way to lay out.  Responsive lay out is implement with flex-boxes.

Reply:
It's not so critical in this task. But I mean why did you separate rendering logic of cups and text information in different sections. Maybe I was wrong that I left this comment in the layout section. I left a comment about it [here](https://github.com/kottans/frontend-2021-homeworks/pull/78/files#r519664676)

Reply:
Now I got it. In my plan was to do function ```toggleContent``` more multi purpose, because render of content cups & articles uses the same code, but ok, let it be as you want.

Reply:
Excuse for this add : I thought that I've written the same part of code for two actions (renders) and I had to provide a function to avoid a duplicate of code.